frontend/src/pages/Extract/index.js

```diff
@@ -6,47 +6,82 @@ import './styles.css'
 import api from '../../service/api';
 import extractGif from '../../assets/extract.gif'
 import Hamburguer from '../../components/Hamburguer'
-import axios from 'axios';
 
 export default function Extract() {
         const [extract, setExtract] = useState([]);
         const [date, setDate] = useState('');
 
-        function HandleExtract(props){
+        function HandleExtractFirstTime(props){
             useEffect(()=>{
                 try {
-                    api.get('/transactions', { params: { user_id: localStorage.getItem('userId'), year_month: '2020-10' } }).then(res => {
-                        setExtract(res.data);
+                    var today = new Date();
+                    var currentDate;
+                    let currentMonth = today.getMonth()+1;
+                    let currentYear = today.getFullYear();
+
+                    if(today.getMonth() >= 9){
+                        currentDate = currentYear + '-0' + currentMonth;
+                    }else{
+                        currentDate = currentYear + '-' + currentMonth;
+                    }         
```

Comment:
As condições estão invertidas: deveria coloca o `-0`para meses menores que 10.

```suggestion
                    if(today.getMonth() < 10 ){
                        currentDate = currentYear + '-0' + currentMonth;
                    }else{
                        currentDate = currentYear + '-' + currentMonth;
                    }         
```

Reply:
Verdade mano. Acabei vendo que não precisava mais dessa condição, então tirei.